OC4IDS_Quality_Criteria_Checks_and_Metrics.ipynb

```diff
@@ -0,0 +1,1792 @@
+{
```

Comment:
<p>This is returning an empty table which means all the other checks also return an empty table, not sure what I'm doing wrong? I used 'dev_1' and 'dev_2' as the default load and comparison_load ids and then also tried with 2 collections I created myself using the import notebook with the same empty result.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-contracting/notebooks-oc4ids/pull/43/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>colab shows this cell as returning an empty table, but the insert command works as expected. </p><p> </p><p>I can see the run_collection table has been populated with the run_id and collections corresponding to dev_1 on the date and time Jen worked on this. </p>